### PR TITLE
WFCORE-4702 Add deployment phases for MicroProfile Fault Tolerance

### PR DESCRIPTION
**Pull request by rhusar**
https://issues.jboss.org/browse/WFCORE-4702

### CODE REVIEW

**Comment by rsvoboda** [comment]
@msve seems MP FT is in move

File: server/src/main/java/org/jboss/as/server/deployment/Phase.java [hunk 1]
```diff
@@ -446,6 +446,7 @@ public Phase next() {
     public static final int DEPENDENCIES_MICROPROFILE_METRICS           = 0x1860;
     public static final int DEPENDENCIES_MICROPROFILE_HEALTH            = 0x1870;
     public static final int DEPENDENCIES_MICROPROFILE_OPENTRACING       = 0x1880;
+    public static final int DEPENDENCIES_MICROPROFILE_FAULT_TOLERANCE   = 0x1890;
```
Comment (jmesnil) [inline server/src/main/java/org/jboss/as/server/deployment/Phase.java]: @rhusar could you bump the constants please? I'm going to merge #3963 that takes these values.

Comment (rhusar) [reply server/src/main/java/org/jboss/as/server/deployment/Phase.java]: Done; would have needed a rebase anyway.

**Comment by rhusar** [comment]
@jmesnil This has been updated as you requested and rebased so this is ready (I am realizing we probably need to proactively ping here but I assumed there is a notification send by github).

**Comment by rhusar** [comment]
I see there is now an option to ask for re-review, so that is probably what was needed to get proper notifications.

**Comment by jmesnil** [comment]
@rhusar thanks

**Pull request merged**